src/main/java/com/commercetools/sync/commons/models/NonResolvedReferencesCustomObject.java

```diff
@@ -0,0 +1,56 @@
+package com.commercetools.sync.commons.models;
+
+import io.sphere.sdk.products.ProductDraft;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+
+public class NonResolvedReferencesCustomObject {
```

Comment:
I would name it `ProductWithUnResolvedProductReferences`. This class has/knows nothing about CTP CustomObject. We should loosen this coupling of concepts.

Reply:
723572d6bbf71ee61b733d15542162ed09bea278